### PR TITLE
refactor: implemented form validation on ui. removed unwanted codes, …

### PR DESCRIPTION
**Pull request by Nishant-96**
This Pr includes refactoring. removed unwanted code, showing update note while editing in create note component. Form validation on ui.

### CODE REVIEW

File: src/context/auth/auth-context.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ function AuthProvider({ children }) {
         });
       }
     } catch (error) {
-      console.log(error);
-      toast.error(`Login Error !`, {
+      console.error();
```
Comment (znjs) [inline src/context/auth/auth-context.js]: Empty console has no inference

File: src/pages/auth/login.jsx [hunk 2]
```diff
@@ -1,13 +1,38 @@
...
+  const loginCLickHandler = (userDetails) => {
+    const validation = formValidationLogin({ ...userDetails });
+    try {
+      if (validation.type) {
+        loginHandler(userDetails.email, userDetails.password);
+      } else {
+        throw new Error(validation.message);
+      }
+    } catch (error) {
+      console.error(error.message);
```
Comment (znjs) [inline src/pages/auth/login.jsx]: console not required if toast is available

**Review by znjs — APPROVED** [review]
- Good Pr
- good naming conventions used

**Review by shobit1337 — APPROVED** [review]
Code looks clean, Good work!

**Pull request merged**